internal/commands/logs/list_test.go

```diff
@@ -386,3 +386,24 @@ func TestLogTypeDisplay(t *testing.T) {
 	}
 	t.Logf("the max width for log type display is: %d", maxWidth)
 }
+
+func TestLogTypeFormat(t *testing.T) {
+	t.Run(fmt.Sprintf("should fail if log type %s is not formatted before passed to client", logTypeSchema), func(t *testing.T) {
```

Comment:
considering we aren't actually asserting any failures here, maybe we re-word this test description?

`with schema log type set should query for all types of schema logs`  (<-- I don't love this either, but you get the idea)